packages/app/src/app/pages/NewDashboard/Content/routes/Repositories/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useParams } from 'react-router-dom';
+import { useOvermind } from 'app/overmind';
+import { Header } from 'app/pages/NewDashboard/Components/Header';
+import { VariableGrid } from 'app/pages/NewDashboard/Components/VariableGrid';
+import { DashboardGridItem, PageTypes } from 'app/pages/NewDashboard/types';
+import { SelectionProvider } from 'app/pages/NewDashboard/Components/Selection';
+import { getPossibleTemplates } from '../../utils';
+import { useFilteredItems } from './useFilteredItems';
+
+export const RepositoriesPage = () => {
+  const params = useParams<{ path: string }>();
+  const items = useFilteredItems(params);
+  const param = params.path || '';
+  const home = !param || param === '/';
+  const {
+    actions,
+    state: {
+      activeTeam,
+      dashboard: { sandboxes, viewMode },
+    },
+  } = useOvermind();
+
+  React.useEffect(() => {
+    const p = home ? null : param;
+    actions.dashboard.getReposByPath(p);
+  }, [param, actions.dashboard, activeTeam, home]);
+
+  const activeSandboxes =
+    (sandboxes.REPOS && Object.values(sandboxes.REPOS)) || [];
+
+  const itemsToShow = (): DashboardGridItem[] => {
+    if (!sandboxes.REPOS) {
```

Comment:
We need to check explicitly with `== null` here, because it now will also show the skeleton in case `sandboxes.REPOS = []` I believe.

Reply:
changed :)